src/Native/pkg/runtime.native.System.Data.SqlClient.sni/win/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <Version>4.0.1</Version>
+    <!-- use the same naming convention as a runtime package, but don't treat as a runtime dependency -->
+    <IdPrefix>runtime.win7-$(PackagePlatform).</IdPrefix>
+  </PropertyGroup>
+
+  <!-- These paths are not built in CoreFx, enable when fixing https://github.com/dotnet/corefx/issues/826 -->
+  <ItemGroup Condition="Exists('$(OutputRootPath)ProjectK')">
```

Comment:
Can you please use the same property conventions we use in other shim packages?
https://github.com/dotnet/corefx/blob/master/src/Native/pkg/dir.props#L6

Add one for windows and have our TFS build override it.

Also, please don't condition File elements.  All shims build with a flag that says to skip packing if the file is absent.
https://github.com/dotnet/corefx/blob/master/src/packages.builds#L6